### PR TITLE
Sync PageLoader

### LINKED ISSUE #44: [Syncing] PageLoader
## Points to note
- Component in action - `PageLoader.js`
- Github Urls to this component:
  * [neeto-ui/PageLoader.js](https://github.com/bigbinary/neeto-ui/tree/67ee4a5188c5feccbeca2767107651fb24932764/lib/components/PageLoader.js)
  * [NitroUI/PageLoader.js](https://github.com/bigbinary/neeto-help-web/tree/0b11a7b0f98ac7c533df1e9ec37cf900b4cbc901/app/javascript/Admin/Components/NitroUI/PageLoader.js)
- :green_circle: Green color signifies how that particular line is written in `neeto-ui` package
- :red_circle: Red color signifies how that particular line is written in `NitroUI` internals

## Sync Required or Not
- Are there significant changes for `PageLoader.js` that needs to be synced to `neeto-ui` package from `NitroUI`? - **No**

## Diff
```diff
--- NitroUI/PageLoader.js	2020-11-28 16:54:23.996346400 +0530
+++ neetoui-official/PageLoader.js	2020-11-28 16:49:18.748010500 +0530
@@ -5,7 +5,7 @@
   return (
     <div className="relative w-full h-full">
       <div className="absolute top-0 left-0 flex flex-row items-center justify-center w-full h-full">
-        <BeatLoader size={16} color={"#252f3f"} />
+        <BeatLoader size={16} color={"#5469D4"} />
       </div>
     </div>
   );
```

### PR DESCRIPTION
**Pull request by yedhink**
Fixes #44 

Just pulled out the spinner color attr into a separate prop and made the default color as specified in neeto-help.

@goutham-subramanyam _a
Please review and merge if everything looks good. Thanks

### CODE REVIEW

File: lib/components/PageLoader.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from "react";
 import { BeatLoader } from "react-spinners";
 
-const PageLoader = () => {
+const PageLoader = ({ spinnerColor = "#252f3f" }) => {
```
Comment (goutham-subramanyam) [inline lib/components/PageLoader.js:4]: We can just call it `color`

**Review by goutham-subramanyam — COMMENTED** [review]
@yedhink _a

**Comment by yedhink** [comment]
@goutham-subramanyam _a

I have made the required changes. Please review and merge if everything looks good.

**Review by goutham-subramanyam — APPROVED** [review]
LGTM.

**Pull request merged**